### PR TITLE
Absolute pathing in pills.dm , firstaid.dm , dna_injector.dm

### PR DESCRIPTION
**Pull request by Probe1**
~~0% tested.~~

~~95%~~ ~~96%~~ 31% tested good enough for me.  At least I tested the first file mostly.

**Ready for merge**

### CODE REVIEW

**Comment by ghost** [comment]
But 95% tested is MY meme see #8627

**Comment by Probe1** [comment]
OC meme do not steal

**Comment by Probe1** [comment]
oh my fucking lord

File: code/game/objects/items/weapons/storage/firstaid.dm [hunk 1]
```diff
@@ -281,10 +281,20 @@ var/global/list/bottle_colour_choices = list("Blue" = "#0094FF","Dark Blue" = "#
...
+
+/obj/item/weapon/storage/pill_bottle/nanobot
+	name = "Experimental Medication"
+	desc = "Hazardous.  Warranty voided if consumed."
+
+	/obj/item/weapon/storage/pill_bottle/nanobot/New()
+		..()
+		for (var/x = 1 to 5)
+			new /obj/item/weapon/reagent_containers/pill/nanobot(src)
+>>>>>>> Bleeding-Edge
```
Comment (Shadowmech88) [inline code/game/objects/items/weapons/storage/firstaid.dm]: This shouldn't be here.

File: code/game/objects/items/weapons/storage/firstaid.dm [hunk 2]
```diff
@@ -207,87 +205,69 @@ var/global/list/bottle_colour_choices = list("Blue" = "#0094FF","Dark Blue" = "#
...
-		..()
-		new /obj/item/weapon/reagent_containers/pill/creatine_safe( src )
-		new /obj/item/weapon/reagent_containers/pill/creatine_supplement ( src )
-		new /obj/item/weapon/reagent_containers/pill/creatine_supplement ( src )
-		new /obj/item/weapon/reagent_containers/pill/creatine_supplement ( src )
-		new /obj/item/weapon/reagent_containers/pill/creatine_supplement ( src )
-		new /obj/item/weapon/reagent_containers/pill/creatine_supplement ( src )
+	..()
+	new /obj/item/weapon/reagent_containers/pill/creatine_safe(src)
+	for (var/x = 1 to 5)
```
Comment (Intigracy) [inline code/game/objects/items/weapons/storage/firstaid.dm]: This is just a nitpick, but could you change these to var/i please, i is the for loop var we use universally.

**Comment by Intigracy** [comment]
I'm assuming he used for var/x as an example or something, I dunno

**Pull request merged**